src/test/java/com/conveyal/osmlib/PostgresTest.java

```diff
@@ -0,0 +1,58 @@
+package com.conveyal.osmlib;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.sql.*;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class PostgresTest {
+
+    @Test
+    public void canLoadFromFileIntoDatabase() {
+        String jdbcUrl = "jdbc:postgresql://localhost/osm_lib_test?user=osm_test&password=osm_test";
+        String[] args = {
+            "./src/test/resources/bangor_maine.osm.pbf",
+            jdbcUrl
+        };
+
+        // perform file to postgres load
+        PostgresSink.main(args);
```

Comment:
I find it kind of strange to be passing string arguments into a main method in a test. Main methods are really only intended to be used as a command line interface.  Ideally we'd pull the "load a feed" functionality out of the main method and call the same functionality from the main and the test. But this will do for now.